dolphin/api/v1/router.py

```diff
@@ -52,7 +52,14 @@ def _setup_routes(self, mapper):
         self.resources['pools'] = pools.create_resource()
         mapper.resource("pool", "pools",
                         controller=self.resources['pools'])
-
+        mapper.connect("storages", "/storages/{storage_id}/pools",
+                       controller=self.resources['pools'],
+                       action="list_pools",
+                       conditions={"method": ["GET"]})
+        mapper.connect("storages", "/storages/{storage_id}/pools/{id}",
+                       controller=self.resources['pools'],
+                       action="show_pool",
+                       conditions={"method": ["GET"]})
```

Comment:
I don't think it's required here, such as storage_id is not required if user knows pool id when get a specific pool. IMO, we can remove the two connects. The capability of /v1/pools and /v1/pools/<id> include them.

Reply:
As discussed , keeping the implemnation now and we will take decision based on API spec review.

Reply:
plz remove it